masonry/src/widget/portal.rs

```diff
@@ -341,8 +354,11 @@ impl<W: Widget> Widget for Portal<W> {
             !self.constrain_vertical && portal_size.height < content_size.height;
 
         if self.scrollbar_horizontal_visible {
-            self.scrollbar_horizontal.widget_mut().portal_size = portal_size.width;
-            self.scrollbar_horizontal.widget_mut().content_size = content_size.width;
+            let mut scrollbar = ctx.get_raw_mut(&mut self.scrollbar_horizontal);
+            scrollbar.widget().portal_size = portal_size.width;
+            scrollbar.widget().content_size = content_size.width;
+            std::mem::drop(scrollbar);
```

Comment:
Do we need to request a paint on the scrollbar here?

Reply:
LayoutCtx currently doesn't have a request_paint method. I'm choosing to procrastinate on that.